### DISCUSSION TITLE
I would like to show off Pixelitor's filters in artistic applications.

### DISCUSSION THREAD

**Discussion by LinuxBeaver** [lbalazscs/Pixelitor#206] [Show and tell]
Hello, I have been making lots of art primarily in Gimp. However Pixelitor is usually evoked to make high quality background design and textures that Gimp cannot do. I am making this thread just to show what Pixelitor can do. I will label what Pixelitor filters were used in each image.

Crystalize (for background)
![image](https://user-images.githubusercontent.com/78667207/123908653-54a6ab80-d967-11eb-843a-133b70962ba8.png)

Radial Waves (for rays) and Value Noise (for clouds)
![image](https://user-images.githubusercontent.com/78667207/123908720-7142e380-d967-11eb-8cd2-cf3c91c85d23.png)

Caustics 
![image](https://user-images.githubusercontent.com/78667207/123908764-8586e080-d967-11eb-8585-911ff17c0fd5.png)

Starburst (off Canvas trick)
![image](https://user-images.githubusercontent.com/78667207/123908799-93d4fc80-d967-11eb-826b-a09aa2e1bdfa.png)

Plasma  and Radial Waves
![image](https://user-images.githubusercontent.com/78667207/123908916-caab1280-d967-11eb-9f59-d35eb1e7a74a.png)

Weave
![image](https://user-images.githubusercontent.com/78667207/123908941-d39be400-d967-11eb-93e7-048e48ef7cd1.png)

Grid with the new fish scales
![image](https://user-images.githubusercontent.com/78667207/123908975-e31b2d00-d967-11eb-9c4c-fca1aa0a33da.png)

Caustics, Value Noise and Grid Fish scales
![image](https://user-images.githubusercontent.com/78667207/123909022-f4643980-d967-11eb-842e-47db7078bfda.png)

Starburst
![image](https://user-images.githubusercontent.com/78667207/123909060-ff1ece80-d967-11eb-8f85-e2e89bde9b1f.png)

Sparkle Inverted and checkerboard 
![image](https://user-images.githubusercontent.com/78667207/123909096-0cd45400-d968-11eb-9dac-d95a4737dd77.png)

Plasma 
![image](https://user-images.githubusercontent.com/78667207/123909124-18c01600-d968-11eb-997f-e3a2e698b72e.png)

Radial Waves
![image](https://user-images.githubusercontent.com/78667207/123909142-24134180-d968-11eb-8b79-75f2c6ff60fc.png)

Gaussian blur and Glass tiles
![image](https://user-images.githubusercontent.com/78667207/123909236-3b522f00-d968-11eb-974b-458401cbd672.png)

Cells and Four Color Gradient
![image](https://user-images.githubusercontent.com/78667207/123909313-5624a380-d968-11eb-914a-2197fb233644.png)

Starburst and Swirl Pinch and Whirl
![image](https://user-images.githubusercontent.com/78667207/123909392-7d7b7080-d968-11eb-979b-3a4ca65798e0.png)

Starburst and Swirl Pinch and Whirl
![image](https://user-images.githubusercontent.com/78667207/123909522-b1ef2c80-d968-11eb-872c-39f0e4e60ef6.png)

Cloud Emboss and a manual color overlay
![image](https://user-images.githubusercontent.com/78667207/123909562-c03d4880-d968-11eb-90f2-560d4bf2e169.png)

Lissajous curve
![image](https://user-images.githubusercontent.com/78667207/123909682-e06d0780-d968-11eb-94ff-180d8e777c99.png)

Plasma and Swirl Pinch Whirl
![image](https://user-images.githubusercontent.com/78667207/123910390-e44d5980-d969-11eb-8431-a93d305a8e72.png)

Once again. Gimp was primarily involved in making sure these filters can be touched up with other things but without Pixelitor it would be much more difficult to make starburst, fish scales, spirals, caustics, glass tiles and other things. This software goes a long way to help Gimp and open source design.

**Comment by lbalazscs** [comment]
Very cool images! Keep them coming! It's OK to also use other software, and not just Pixelitor.

**Comment by LinuxBeaver** [comment]
Starburst
![image](https://user-images.githubusercontent.com/78667207/125379542-389c0480-e380-11eb-932e-bc267c523dd1.png)

Starburst and Halftone
![image](https://user-images.githubusercontent.com/78667207/125379561-42256c80-e380-11eb-9b29-7df1f611f96e.png)

Crystalize and Cells
![image](https://user-images.githubusercontent.com/78667207/125380474-c1677000-e381-11eb-9cec-b71c633b5b94.png)

**Comment by Minecraftian14** [comment]
Not the best, definately....
But some good ideas for creating nice backgrounds 😁

Fill the scene with a nice collection of colors smoothening towards each other; finally smudging the whole image starting from the center and moving outwards in a spiral like manner.
Here I lazily used Color Wheel. 
![image](https://user-images.githubusercontent.com/52451860/125770435-371117be-9bf1-4b68-86d9-ff3974c80578.png)

With a similar background, create a new layer and set it's blending mode to `Soft Light`. 
Pick a nice brush and apply `File>Automate>Auto Paint`
I used Four Color Gradient.
![image](https://user-images.githubusercontent.com/52451860/125771181-2bd51d26-d7a1-41dc-809c-d5a84ec7435f.png)

Ah... i like this background soooo much for testing out things 😁
To get this effect, generate some pretty `clouds` on another layer, and decrease the colors's `transparency`; finally applying `Crystallize`
![image](https://user-images.githubusercontent.com/52451860/125771248-98a2ed61-dd25-47df-a1f5-87afa3e862c5.png)

Pick a nice brush, increase it's size and start tapping the screen to create these nice circles. Every time you create a circle, press R.
![image](https://user-images.githubusercontent.com/52451860/125964675-b256c961-e427-48c8-b72c-a3ff272e040b.png)

Over a 4 color gradient, create another four color gradient in a separate layer with a mask. On the mask, add clouds with translucent colors. Optionally you can add yet another layer with clouds with transparent white and translucent black.
![image](https://user-images.githubusercontent.com/52451860/125964708-4dfbfa35-d0ff-4b24-becf-bb0f71cdf08d.png)

Over a nice dark/black/light/white background, generate marble on a separate layer. Just pick nice bright colors and adjust transparency.
![image](https://user-images.githubusercontent.com/52451860/125964752-d3d45789-560d-4f75-a69b-097a9352a960.png)

Similarly, instead of marble, one can use caustics
![image](https://user-images.githubusercontent.com/52451860/125964736-28615d97-4172-43a7-8fa3-e65ffbed689f.png)

**Comment by LinuxBeaver** [comment]
This filter was released less then twenty four hours ago and I already found a use for it.

![image](https://user-images.githubusercontent.com/78667207/126083096-1956ac72-e8bd-4db4-9b86-9a38ba1d0c92.png)

**Reply by lbalazscs** [reply]
Nice! In the newest commit, I added more Truchet patterns, I hope you can do something with them...

**Comment by LinuxBeaver** [comment]
Testing what I can do now.
![image](https://user-images.githubusercontent.com/78667207/126430025-47c4124b-35c2-4378-8643-07019acf3501.png)

![image](https://user-images.githubusercontent.com/78667207/126430165-2ba178ef-4e2f-46c2-85bd-a4bfa281a9db.png)



![image](https://user-images.githubusercontent.com/78667207/126430256-c58e4422-bcf4-4ec8-a9f3-898b2d4c9257.png)
![image](https://user-images.githubusercontent.com/78667207/126431219-005f20bb-2105-4991-b693-0e81e0487efd.png)

**Comment by LinuxBeaver** [comment]
Truchets and Four Color Gradient.
![image](https://user-images.githubusercontent.com/78667207/126604513-10a75743-41d8-4bb3-aaeb-c04e7801d099.png)

![image](https://user-images.githubusercontent.com/78667207/126604221-687eceb4-9ac8-488f-a202-2ca897f982eb.png)

**Reply by lbalazscs** [reply]
Very nice images! I guess the 3D effect on the background was done by bumpmapping in Gimp. Makes me want to have this feature in Pixelitor.

**Comment by LinuxBeaver** [comment]
You can use this image of my desktop to show off Pixelitor's potential in background design.

![image](https://user-images.githubusercontent.com/78667207/132070263-be5c67f2-c7aa-4dc3-9782-c05c81e6111f.png)

**Comment by lbalazscs** [comment]
It's a very nice image, although I don't do much showing off. Perhaps I could put it on the Pixelitor homepage, instead of the purple whirl, which also shows your desktop...

BTW, the look of Pixelitor could also change in the future, for example with a flat and dark toolkit it could look something like this:
![flatlaf](https://user-images.githubusercontent.com/9939454/132141751-7851adb3-a67d-4419-bdec-58b220472a29.png)

**Comment by LinuxBeaver** [comment]
Once I stumbled upon a half way build of pixelitor and it had the ability to use my system's native theme (Cinnamon Mint Y; based on Arc) It is the default theme for Linux Mint and its Cinnamon Desktop

It crashed the program but I remember finding it interesting.  I myself am not a fan of dark themes but I understand most people like them. I just go with system defaults.

**Comment by LinuxBeaver** [comment]
Glass tiles and Cells
![image](https://user-images.githubusercontent.com/78667207/133910408-9cf23b32-0423-46ed-ba20-f25793d81fb9.png)
